packages/terra-button/src/Button.jsx

```diff
@@ -46,6 +46,10 @@ const propTypes = {
    */
   text: PropTypes.string,
   /**
+   * Sets the button type. One of button, submit, or reset
+   */
+  type: PropTypes.oneOf(['button', 'submit', 'reset']),
```

Comment:
Let's stick with strings for Phase 1 work. As part of the Q3 work, we can look more into using enum objects. 

The enum objects would be an option for consumers to import that should be passive the way we've been doing current prop-types.